src/Resources/skeleton/security/UserProvider.tpl.php

```diff
@@ -3,7 +3,7 @@
 namespace <?= $namespace; ?>;
 
 use Symfony\Component\Security\Core\Exception\UnsupportedUserException;
-use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
+use Symfony\Component\Security\Core\Exception\<?= $uses_user_identifier ? 'UserNotFoundException' : 'UsernameNotFoundException' ?>;
 <?= ($password_upgrader = interface_exists('Symfony\Component\Security\Core\User\PasswordUpgraderInterface')) ? "use Symfony\Component\Security\Core\User\PasswordUpgraderInterface;\n" : '' ?>
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\User\UserProviderInterface;
```

Comment:
We really need to, at some point, write a util class where we plug all the `use` statements into... then we just dump `echo $useStatements->render()` in the template :p

Reply:
No kidding.. use statements are getting harder and harder to manage lately.. have a solution on the todo list ;)